src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4552,326 +4552,181 @@ internal static SessionStateAliasEntry[] BuiltInAliases
-                    new SessionStateAliasEntry("set",
-                        "Set-Variable",    "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("icm",
-                        "Invoke-Command",  "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("clhy",
-                        "Clear-History",    "", ScopedItemOptions.ReadOnly | ScopedItemOptions.AllScope),
+                    new SessionStateAliasEntry("h", "Get-History"),
+                    new SessionStateAliasEntry("history", "Get-History"),
+                    new SessionStateAliasEntry("md", "mkdir", "", AllScope),
+                    new SessionStateAliasEntry("popd", "Pop-Location", "", AllScope),
+                    new SessionStateAliasEntry("pushd", "Push-Location", "", AllScope),
+                    new SessionStateAliasEntry("r", "Invoke-History"),
+                    new SessionStateAliasEntry("cls", "Clear-Host"),
+                    new SessionStateAliasEntry("chdir", "Set-Location"),
+                    new SessionStateAliasEntry("copy", "Copy-Item", "", AllScope),
+                    new SessionStateAliasEntry("del", "Remove-Item", "", AllScope),
+                    new SessionStateAliasEntry("erase", "Remove-Item"),
+                    new SessionStateAliasEntry("move", "Move-Item", "", AllScope),
+                    new SessionStateAliasEntry("rd", "Remove-Item"),
+                    new SessionStateAliasEntry("ren", "Rename-Item"),
+                    new SessionStateAliasEntry("set", "Set-Variable"),
+                    new SessionStateAliasEntry("icm", "Invoke-Command"),
+                    new SessionStateAliasEntry("clhy", "Clear-History", "", ReadOnly),
                     // Job Specific aliases
-                    new SessionStateAliasEntry("gjb",
-                        "Get-Job",  "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("rcjb",
-                        "Receive-Job",  "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("rjb",
-                        "Remove-Job",  "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("sajb",
-                        "Start-Job",  "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("spjb",
-                        "Stop-Job",  "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("wjb",
-                        "Wait-Job",  "", ScopedItemOptions.AllScope),
+                    new SessionStateAliasEntry("gjb", "Get-Job"),
+                    new SessionStateAliasEntry("rcjb", "Receive-Job"),
+                    new SessionStateAliasEntry("rjb", "Remove-Job"),
+                    new SessionStateAliasEntry("sajb", "Start-Job"),
+                    new SessionStateAliasEntry("spjb", "Stop-Job"),
+                    new SessionStateAliasEntry("wjb", "Wait-Job"),
 #if !CORECLR
```

Comment:
Should we remove the !CORECLR section?

Reply:
I thought most of these cmdlets might be ported to CORECLR, so left it.

Reply:
same as above